<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUp</class>
 <widget class="QDialog" name="SignUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label">
           <property name="cursor">
            <cursorShape>ArrowCursor</cursorShape>
           </property>
           <property name="text">
            <string>Name</string>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit">
           <property name="placeholderText">
            <string>First name only</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="cursor">
            <cursorShape>ArrowCursor</cursorShape>
           </property>
           <property name="text">
            <string>Email</string>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_2">
           <property name="placeholderText">
            <string>Acceptable format: ex@ex.com</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="cursor">
            <cursorShape>ArrowCursor</cursorShape>
           </property>
           <property name="text">
            <string>National ID</string>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_6">
           <property name="placeholderText">
            <string>Numbers only</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="cursor">
            <cursorShape>ArrowCursor</cursorShape>
           </property>
           <property name="text">
            <string>Password</string>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_3">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
           <property name="placeholderText">
            <string>No spaces</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="cursor">
            <cursorShape>ArrowCursor</cursorShape>
           </property>
           <property name="text">
            <string>Confirm Password</string>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_4">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="cursor">
            <cursorShape>ArrowCursor</cursorShape>
           </property>
           <property name="text">
            <string>Date of birth</string>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dateEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>75</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Specify wanted account type</string>
         </property>
         <property name="title">
          <string>Account Type</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_13">
            <item>
             <widget class="QRadioButton" name="radioButton">
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="text">
               <string>Donor</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup_3</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_2">
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="text">
               <string>Recipient</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup_3</string>
              </attribute>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>75</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Specify your gender</string>
         </property>
         <property name="title">
          <string>Gender</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_14">
            <item>
             <widget class="QRadioButton" name="radioButton_5">
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="text">
               <string>Male</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup_2</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_6">
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="text">
               <string>Female</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup_2</string>
              </attribute>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_15">
         <item>
          <widget class="QWidget" name="widget" native="true">
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QLabel" name="label_9">
                  <property name="cursor">
                   <cursorShape>ArrowCursor</cursorShape>
                  </property>
                  <property name="text">
                   <string>Doctor of case</string>
                  </property>
                  <property name="textInteractionFlags">
                   <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_7">
                  <property name="toolTip">
                   <string>The first name of the responsible doctor of the case</string>
                  </property>
                  <property name="placeholderText">
                   <string>First name only</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="cursor">
                   <cursorShape>ArrowCursor</cursorShape>
                  </property>
                  <property name="text">
                   <string>Disease</string>
                  </property>
                  <property name="textInteractionFlags">
                   <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_5">
                  <property name="toolTip">
                   <string>The disease you suffer from</string>
                  </property>
                  <property name="placeholderText">
                   <string/>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QLabel" name="label_7">
                  <property name="cursor">
                   <cursorShape>ArrowCursor</cursorShape>
                  </property>
                  <property name="text">
                   <string>Hospital</string>
                  </property>
                  <property name="textInteractionFlags">
                   <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboBox">
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <item>
                   <property name="text">
                    <string>hopital_1</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>hopital_2</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>hopital_3</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>hopital_4</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_2" native="true">
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QLabel" name="label_10">
                  <property name="cursor">
                   <cursorShape>ArrowCursor</cursorShape>
                  </property>
                  <property name="text">
                   <string>Date of last donation</string>
                  </property>
                  <property name="textInteractionFlags">
                   <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDateEdit" name="dateEdit_2"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QLabel" name="label_12">
                  <property name="cursor">
                   <cursorShape>ArrowCursor</cursorShape>
                  </property>
                  <property name="text">
                   <string>Disease</string>
                  </property>
                  <property name="textInteractionFlags">
                   <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboBox_2">
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="toolTip">
                   <string>Select the disease you suffer from</string>
                  </property>
                  <item>
                   <property name="text">
                    <string>Blood pressure disorder</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Thyroid disease</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Diabetes</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Cancer</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Heart disorders</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Hepatitis</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>None</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="label_11">
                  <property name="cursor">
                   <cursorShape>ArrowCursor</cursorShape>
                  </property>
                  <property name="text">
                   <string>Other diseases or medicines</string>
                  </property>
                  <property name="textInteractionFlags">
                   <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_8">
                  <property name="toolTip">
                   <string/>
                  </property>
                  <property name="placeholderText">
                   <string>If none type: &quot;NA&quot;</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>Back to login</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>Signup</string>
         </property>
         <property name="shortcut">
          <string>Return</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Specify your blood type</string>
       </property>
       <property name="title">
        <string>Blood Type</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <widget class="QRadioButton" name="radioButton_14">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>A+</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QRadioButton" name="radioButton_4">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>AB+</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QRadioButton" name="radioButton_3">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>B-</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QRadioButton" name="radioButton_8">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>AB-</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QRadioButton" name="radioButton_12">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>O+</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QRadioButton" name="radioButton_13">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>B+</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QRadioButton" name="radioButton_10">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>O-</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="radioButton_9">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>A-</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>lineEdit_6</tabstop>
  <tabstop>lineEdit_3</tabstop>
  <tabstop>lineEdit_4</tabstop>
  <tabstop>dateEdit</tabstop>
  <tabstop>radioButton</tabstop>
  <tabstop>radioButton_2</tabstop>
  <tabstop>radioButton_5</tabstop>
  <tabstop>radioButton_6</tabstop>
  <tabstop>lineEdit_7</tabstop>
  <tabstop>lineEdit_5</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>dateEdit_2</tabstop>
  <tabstop>comboBox_2</tabstop>
  <tabstop>lineEdit_8</tabstop>
  <tabstop>radioButton_14</tabstop>
  <tabstop>radioButton_9</tabstop>
  <tabstop>radioButton_13</tabstop>
  <tabstop>radioButton_3</tabstop>
  <tabstop>radioButton_4</tabstop>
  <tabstop>radioButton_8</tabstop>
  <tabstop>radioButton_12</tabstop>
  <tabstop>radioButton_10</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>radioButton</sender>
   <signal>clicked()</signal>
   <receiver>widget</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>88</x>
     <y>376</y>
    </hint>
    <hint type="destinationlabel">
     <x>103</x>
     <y>593</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton</sender>
   <signal>clicked()</signal>
   <receiver>widget_2</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>76</x>
     <y>376</y>
    </hint>
    <hint type="destinationlabel">
     <x>902</x>
     <y>593</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_2</sender>
   <signal>clicked()</signal>
   <receiver>widget</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>376</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>593</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_2</sender>
   <signal>clicked()</signal>
   <receiver>widget_2</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>376</y>
    </hint>
    <hint type="destinationlabel">
     <x>902</x>
     <y>593</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroup_3"/>
  <buttongroup name="buttonGroup_2"/>
 </buttongroups>
</ui>
